modules/gapi/src/streaming/onevpl/accelerators/accel_policy_dx11.hpp

```diff
@@ -0,0 +1,66 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef GAPI_STREAMING_ONEVPL_ACCELERATORS_ACCEL_POLICY_DX11_HPP
+#define GAPI_STREAMING_ONEVPL_ACCELERATORS_ACCEL_POLICY_DX11_HPP
+
+#include "opencv2/gapi/own/exports.hpp" // GAPI_EXPORTS
+//TODO
```

Comment:
What?

Reply:
would be changed on `//TODO Remove the next MACRO right after DX11 implementation`